kit/ChildSession.cpp

```diff
@@ -1639,6 +1644,35 @@ bool ChildSession::mouseEvent(const StringVector& tokens,
     return true;
 }
 
+bool ChildSession::readOnlyClickEvent(const StringVector& tokens)
+{
+    getLOKitDocument()->setView(_viewId);
+
+    if (tokens.size() != 3)
+        return false;
+
+    const int x = Util::safe_atoi(tokens[1].data(), tokens[1].size());
+    const int y = Util::safe_atoi(tokens[2].data(), tokens[2].size());
+    char* test = getLOKitDocument()->hyperlinkInfoAtPosition(x, y);
+
+    if (test != nullptr)
+    {
+        std::string result(test);
+        std::free(test);
+
+        if (!result.empty() && x != 0 && y != 0)
+        {
+            std::stringstream message;
```

Comment:
`std::ostringstream` is better, as it only allows for writing. This protects against accidental reading, instead of writing.